web/themes/custom/contribtracker/package.json

```diff
@@ -3,11 +3,15 @@
     "build": "gulp",
     "lint": "CI=true gulp lint",
     "vr": "percy exec -- cypress run",
-    "cypress-open": "cypress open --browser chrome",
-    "cypress-run": "cypress run --browser chrome",
-    "cypress:run:without-video": "CYPRESS_VIDEO=false cypress run"
+    "cypress:open": "cypress open --browser chrome",
+    "cypress:run": "cypress run --browser chrome",
+    "cypress:run:without-video": "CYPRESS_VIDEO=false cypress run",
+    "test:smoke": "cypress run --env grepTags=smoke",
+    "test:regression": "cypress run --env grepTags=regression",
+    "test:expensive": "cypress run --env grepTags=expensive"
```

Comment:
@Yemaneberhan-Lemma, it's also inconsistent that some of these have `cypress` as prefix and others have `test`. Is there a reason for different prefix?

Reply:
Fixed.